measurement/units.py

```diff
@@ -9,6 +9,7 @@ class NetworkUnit(Enum):
     megabit_per_second = "Mbit/s"
     kibibit_per_second = "Kibit/s"
     mebibit_per_second = "Mibit/s"
+    byte_per_second = "Byte/s"
```

Comment:
Any reason you've gone with `byte_per_second` here?

It's standard practice to measure transfer rates in `bit/s`, we already have that primitive unit available as `bit_per_second` (plus all of its derivatives) and the various other measurements that report network rates do so in bits rather than bytes.

Reply:
I went with bytes to avoid as many conversions as possible, bits/s makes more sense with the way we present the rest of the data though.

Reply:
Yeah it's a fine/confusing line. I'd say a `/ 8` for fundamental bytes -> bits conversion internal to the measurement is acceptable. It's when we start converting to kilo/kibi/mega/mebi bits/Bytes that things get a bit mental.